Makefile

```diff
@@ -187,6 +198,13 @@ kind-controller-all-kubeconfigs:
 	CLUSTER_NAME=$(HUB_CLUSTER_NAME) KIND_NAME=$(KIND_HUB_NAME) KIND_CLUSTER_NAME=$(KIND_HUB_CLUSTER_NAME) $(MAKE) kind-controller-kubeconfig
 	yq e '.clusters[0].cluster.server = "https://$(KIND_HUB_NAME)-control-plane:6443"' $(HUB_CONFIG) > $(HUB_CONFIG_INTERNAL)
 
+.PHONY: kind-deploy-controller-dev-addon
+kind-deploy-controller-dev-addon:
+	@echo Hosted mode test
+	kind load docker-image $(REGISTRY)/$(IMG):$(TAG) --name $(KIND_NAME)
+	kubectl annotate -n $(subst -hosted,,$(KIND_NAMESPACE)) --overwrite managedclusteraddon governance-policy-framework\
+		addon.open-cluster-management.io/values='{"global":{"imagePullPolicy": "Never", "imageOverrides":{"governance_policy_framework_addon": "$(REGISTRY)/$(IMG):$(TAG)"}}}'
```

Comment:
```suggestion
.PHONY: kind-deploy-controller-dev-addon
kind-deploy-controller-dev-addon:
	@echo Hosted mode test
	kind load docker-image $(REGISTRY)/$(IMG):$(TAG) --name $(KIND_NAME)
	kubectl annotate -n $(subst -hosted,,$(KIND_NAMESPACE)) --overwrite managedclusteraddon governance-policy-framework\
		addon.open-cluster-management.io/values='{"global":{"imagePullPolicy": "Never", "imageOverrides":{"governance_policy_framework": "$(REGISTRY)/$(IMG):$(TAG)"}}}'
```

Reply:
I cherry-picked this commit from stolostron in order to fix the hosted mode tests. Is there a reason we would use `governance_policy_framework` over the `framework_addon`?

Reply:
Oh I checked addon-controller governance_policy_framework_addon should be right I should change framework-test! Good!